inst/traduire/en-translation.json

```diff
@@ -116,5 +116,11 @@
     "SCOPE_AREAS_MISSING_HIERARCHY": "Scope area {{missing_areas}} not found in hierarchy.",
     "SCOPE_AREAS_MISSING_HIERARCHY_PLURAL": "Scope areas {{missing_areas}} not found in hierarchy.",
     "SHAPE_SPECTRUM_REGION_ALL_NA": "Spectrum region code column is all missing in your shape file. Update the shape file with Spectrum region code for the estimation level.",
-    "INVALID_CALIBRATE_OBJECT": "Can't calibrate this model output please re-run model and try calibration again"
+    "INVALID_CALIBRATE_OBJECT": "Can't calibrate this model output please re-run model and try calibration again",
+    "ANC_CLIENTS": "ANC clients",
+    "ANC_PLHIV": "HIV positive ANC attendees",
+    "ANC_ALREADY_ART": "ANC attendees already on ART",
+    "INDICATOR_LABEL_ANC_CLIENTS": "Total number attending first ANC",
+    "INDICATOR_LABEL_ANC_PLHIV": "Number of HIV positive women attending ANC",
+    "INDICATOR_LABEL_ANC_ALREADY_ART": "Number already on ART prior prior to first ANC"
```

Comment:
Additions to the English dictionary. French translations needed.

Reply:
Crude translations added to French dictionary from Google translate. Created issue #169 to check these.